README.mdown

```diff
@@ -1,131 +1,211 @@
-# Generate Dependent Styles
-The script `generate_styles.rb` can be used to automatically generate dependent CSL styles for publishers that use the same citation format(s) across multiple journals. All that is needed is a collection of journal metadata (journal title, ISSN, eISSN, citation format, etc.).
+#### Notice
+This repository was created on 2016-03-10 as a dedicated home for the journal metadata previously stored at https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles.
```

Comment:
Good point. How about 
"previously stored at https://github.com/citation-style-language/utilities/" (in the now-removed "generate_dependent_styles" subdirectory)".